WebContent/zip-fs.js

```diff
@@ -104,7 +104,7 @@
 				child.reader = new child.Reader(child.data, onerror);
 				child.reader.init(function() {
 					child.uncompressedSize = child.reader.size;
-					next();
+					setTimeout(next, 1);
```

Comment:
Use `nextTick` (with Promises when available) as I suggested, and document why we are using it.

Otherwise the perceived performance of the code may drop significantly when there are many files.

`setTimeout` posts a macrotask, `Promise.resolve().then` a microtask. For the difference, see https://stackoverflow.com/questions/25915634/difference-between-microtask-and-macrotask-within-an-event-loop-context

Reply:
Changed. Please check and resolve